app/controllers/api/v2/base_controller.rb

```diff
@@ -64,6 +65,15 @@ def metadata_total
         @total ||= resource_scope.try(:count).to_i
       end
 
+      def can_create
+        kontroller = controller_name.singularize.capitalize.constantize
+        if kontroller.try(:ancestors).include?(Authorizable)
```

Comment:
you can just check `include?` directly on the class, no need to get the ancestors. (https://ruby-doc.org/core-2.3.0/Module.html#method-i-include-3F)